documentation/docs/src/clients/socks5-client.md

```diff
@@ -99,19 +102,22 @@ The `--provider` field needs to be filled with the Nym address of a Network Requ
 Since the nodes on this list are the infrastructure for [Nymconnect](https://nymtech.net/developers/quickstart/nymconnect-gui.html) they will support all apps on the [default whitelist](../nodes/network-requester-setup.md#network-requester-whitelist): Keybase, Telegram, Electrum, Blockstream Green, and Helios.
 
 #### Choosing a Gateway
+
 By default - as in the example above - your client will choose a random gateway to connect to.
 
 However, there are several options for choosing a gateway, if you do not want one that is randomly assigned to your client:
+
 * If you wish to connect to a specific gateway, you can specify this with the `--gateway` flag when running `init`.
 * You can also choose a gateway based on its location relative to your client. This can be done by appending the `--latency-based-selection` flag to your `init` command. This command means that to select a gateway, your client will:
-	* fetch a list of all availiable gateways
+	* fetch a list of all available gateways
 	* send few ping messages to all of them, and measure response times.
-	* create a weighted distribution to randomly choose one, favouring ones with lower latency.
+	* create a weighted distribution to randomly choose one, favoring ones with lower latency.
```

Comment:
this isnt a typo fix but is just changing the spelling from british to american english. Since the docs are written in british english im not going to accept this change.